docs/server/source/quickstart.md

```diff
@@ -8,7 +8,7 @@ A. Install MongoDB as the database backend. (There are other options but you can
 
 B. Run MongoDB. Open a Terminal and run the command:
 ```text
-$ mongod --replSet=bigchain-rs
+$ sudo mongod --replSet=bigchain-rs
```

Comment:
@r-marques : I think MongoDB needs sudo to chown & chgrp to 'mongod' during boot.
That's the default setup, which is what happens in docker too.